### PR TITLE
Disable grading emails on account creation

### PR DESCRIPTION
**Pull request by rshipp**
### Description

Disable grading emails by default when creating new user accounts. 

### Motivation and Context

Task: https://app.asana.com/0/1174274412967132/1199978731682880/f

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have added neccessary labels
- [ ] My PR title follows the convention in the guide
- [ ] I have assigned myself to the PR
- [ ] I have requested reviews from team members
- [ ] I have created a task for reviewers on PM tool
- [ ] All new and existing tests passed.

### CODE REVIEW

File: lib/canvas_api.rb [hunk 1]
```diff
@@ -166,6 +166,32 @@ def find_user_in_canvas(search_term)
     users.length == 1 ? users[0] : nil
   end
 
+  # https://canvas.instructure.com/doc/api/communication_channels.html
+  def get_user_communication_channels(user_id)
```
Comment (sadleb) [inline lib/canvas_api.rb:170]: Can this API call return a paged response? I assume no bc the returned objects are small and there aren’t going to be that many channels? Something to double check and used the get paged response helper if so

File: app/services/register_user_account.rb [hunk 2]
```diff
@@ -81,6 +86,17 @@ def create_canvas_user!
     @new_user.update!(canvas_user_id: canvas_user['id'])
   end
 
+  def update_canvas_user_settings
+    # Note: Several Canvas API calls are wrapped in this one method.
+    CanvasAPI.client.disable_user_grading_emails(@new_user.canvas_user_id)
+
+  rescue RestClient::Exception => e
```
Comment (sadleb) [inline app/services/register_user_account.rb:93]: May as well log it with the logger and use Sentry.capture exception as well so it’s in all of our logging / monitoring systems

File: lib/canvas_api.rb (hunk 1 above)
Comment (rshipp) [reply lib/canvas_api.rb:170]: It does, I'll update to use the helper.

**Review by sadleb — APPROVED** [review]

File: spec/factories/canvas_communication_channel.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    initialize_with { attributes.stringify_keys }
+  end
+end
+
+# Example
+# {
+#  // The ID of the communication channel.
+#  "id": 16,
+#  // The address, or path, of the communication channel.
+#  "address": "sheldon@caltech.example.com",
```
Comment (sadleb) [inline spec/factories/canvas_communication_channel.rb:25]: Big Bang theory reference? 😂

**Pull request merged**